lib/MetaCPAN/Model/Search.pm

```diff
@@ -44,6 +44,16 @@ sub search_simple {
     return $results;
 }
 
+sub search_first {
```

Comment:
In that case maybe even just `search_for_first_result` would convey what it's doing a little more clearly?